### DISCUSSION TITLE
Multi Github account

### DISCUSSION THREAD

**Discussion by xt-avenirdata** [community/community#30553] [Copilot Conversations]
Hello, I have several accounts on github (Personal, Company, and Personal but for school) 

If I want to use github on two of my accounts ( Personal / Personal but for school ) I have to pay twice the bill ? 200€ in total ... even 300 if I want to use it in a professional context ...  

Is there no method for multiple account service, because I am the same person on all accounts ? ????

**Comment by iamcjwilliams** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
